src/scripts/find-successful-workflow.js

```diff
@@ -92,9 +92,20 @@ function commitExists(commitSha) {
   }
 }
 
-async function isWorkflowSuccessful(pipelineId) {
-  return getJson(`https://circleci.com/api/v2/pipeline/${pipelineId}/workflow`)
-    .then(({ items }) => items.every(item => item.status === 'success'));
+async function isWorkflowSuccessful(pipelineId, workflowName) {
+  if (!workflowName || workflowName.length === 0) {
```

Comment:
`!workflowName` and `workflowName.length === 0` are equivalent for strings.
Use just the first condition.